rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -487,24 +487,32 @@ def get_and_decode_undecoded_transactions(
             self,
             limit: Optional[int] = None,
             addresses: Optional[list[ChecksumEvmAddress]] = None,
+            send_ws_notifications: bool = False,
     ) -> None:
         """Checks the DB for up to `limit` undecoded transactions and decodes them.
         If a list of addresses is provided then only the transactions involving those
         addresses are decoded.
 
         This is protected by concurrent access from a lock"""
         with self.undecoded_tx_query_lock:
+            log.debug(f'Starting task to process undecoded transactions for {self.evm_inquirer.chain_name} with {limit=}')  # noqa: E501
             hashes = self.dbevmtx.get_transaction_hashes_not_decoded(
                 chain_id=self.evm_inquirer.chain_id,
                 limit=limit,
                 addresses=addresses,
             )
-            self.decode_transaction_hashes(ignore_cache=False, tx_hashes=hashes)
+            self.decode_transaction_hashes(
+                ignore_cache=False,
+                tx_hashes=hashes,
+                send_ws_notifications=send_ws_notifications,
+            )
+            log.debug(f'Finished task decoding transactions for {self.evm_inquirer.chain_name} with {limit=}')  # noqa: E501
```

Comment:
Log needs to start with same phrase. Easy to grep/search for start/finish. Otherwise you need to tweak.

```suggestion
            log.debug(f'Finished task to process undecoded transactions for {self.evm_inquirer.chain_name} with {limit=}')  # noqa: E501
```